vyper/codegen/expr.py

```diff
@@ -182,7 +182,7 @@ def parse_Name(self):
 
             ofst = varinfo.position.offset
 
-            if self.context.sig.is_init_func:
+            if self.context.sig.is_constructor:
```

Comment:
note: this will need to change once https://github.com/vyperlang/vyper/pull/3388 is merged

Reply:
will await the merge conflict :smiley: